routing/matcher_test.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/zalando/skipper/eskip"
 	"github.com/zalando/skipper/pathmux"
+	"github.com/zalando/skipper/predicates/methods"
```

Comment:
@aryszka this causes a import cycle:
```
# github.com/zalando/skipper/routing
import cycle not allowed in test
package github.com/zalando/skipper/routing (test)
	imports github.com/zalando/skipper/predicates/methods
	imports github.com/zalando/skipper/routing
```

Reply:
you would need to move routing.Predicate and routing.PredicateSpec to predicates package